Finmer.Editor/FormDocumentScene.cs

```diff
@@ -510,6 +510,49 @@ private void trvNodes_DragDrop(object sender, DragEventArgs e)
             Dirty = true;
         }
 
+        public TreeNode GetTreeNodeByKey(string key) 
+        {
+            //Prepare stack with root TreeNodes in it
```

Comment:
A space in between the `//` and the comment, please. (Same for the other comments below.)